data/templates/esx-ks

```diff
@@ -166,8 +184,12 @@ cp /var/log/esxi_install.log "/vmfs/volumes/datastore1/firstboot-esxi_install.lo
   <% }); %>
 <% } %>
 
-#signify ORA the installation completed
+# signify ORA the installation completed, the 60s sleep is to wait for the LRU to timeout
+# the stale entries that it latched prior to the lookups loading above being set.  Without
+# the sleep, the completionUri hit below will not publish to the correct AMQP channel
+sleep 60
```

Comment:
This is a temporary workaround for 1.2 branch to avoid changing the POST /lookups route to use something else in on-core that would invalidate the LRU entries when the entry is created directly in waterline.